### ISSUE TITLE
during training i got validation loss in negative ?

### ISSUE THREAD

**Issue by xiaoxiao12323** [JK-the-Ko/SAR-CAM#7]
i got negative validation loss after training ? could you tell me how the model compare images. i have low and high imaes pair as a input images. i consider low as a noisy and high quality images as clean image. 
![image](https://github.com/JK-the-Ko/SAR-CAM/assets/171214044/cc8dbb95-99fb-423f-9867-6aa5150998f1)

**Comment by JK-the-Ko** [comment]
As stated in the paper, our proposed loss function is in log scale. Hence, a perfectly optimized model gives you a negative infinite value.
Please refer to the paper.
